tools/ocp-check-headers/checkHeaders.ml

```diff
@@ -119,31 +140,50 @@ let new_header env header_sep header_pos header_lines  =
     in
     [header_pos, h]
 
+let has_magic_word =
+  let regexp =
+    String.concat "\\|"
+      (List.map Str.quote
+         [ "copyright" ; "Copyright" ;
+           "copyleft" ; "Copyleft" ;
+           "(c)" ; "(C)" ; "©" ;
+           "License" ; "license" ; "LICENSE" ;
```

Comment:
You can modify the `matches` function to use `Str.regexp_case_fold` instead of `Str.regexp`.